templates/participants.html

```diff
@@ -72,7 +72,11 @@ <h1>Wszyscy uczestnicy</h1>
                   {{ person.has_letter | qualified_mark }}
                 </td>
                 <td>
-                  {{ person.status }}
+                  {% if person.status == 'Z' %}
+                    <span class="qualified">Zakwalifikowany</span>
```

Comment:
When I said "do it like it's done in the workshops" I meant `{{ person.get_status_display | default_if_none:"Brak" }}`. If you really want colors (which I actually like :P) add them in both, but don't repeat the strings that are already defined in Django.